src/behavior-considered-undefined.md

```diff
@@ -26,8 +26,21 @@ code.
 * Evaluating a [dereference expression] (`*expr`) on a raw pointer that is
   [dangling] or unaligned, even in [place expression context]
   (e.g. `addr_of!(&*expr)`).
-* Breaking the [pointer aliasing rules]. `&mut T` and `&T` follow LLVM’s scoped
-  [noalias] model, except if the `&T` contains an [`UnsafeCell<U>`].
+* Breaking the [pointer aliasing rules]. `Box<T>`, `&mut T` and `&T` follow
+  LLVM’s scoped [noalias] model, except if the `&T` contains an
+  [`UnsafeCell<U>`]. References and boxes must not be dangling while they are
```

Comment:
What *exactly* does "dangling" mean here? What does this allow and disallow?

Reply:
Could this please forward-reference `#dangling-pointers` with a link?

Reply:
Good point, done.